pandas/core/tools/numeric.py

```diff
@@ -108,6 +110,21 @@ def to_numeric(arg, errors="raise", downcast=None):
     2    2.0
     3   -3.0
     dtype: float64
+
+    Downcasting of nullable integer and floating dtypes is supported:
+
+    >>> s = pd.Series([1, 2, 3], dtype="Int64")
+    >>> pd.to_numeric(s, downcast="integer")
+    0    1
+    1    2
+    2    3
+    dtype: Int8
+    >>> s = pd.Series([1.0, 2.1, 3.0], dtype="Float64")
+    >>> pd.to_numeric(s, downcast="float")
```

Comment:
we *may* want to also accept `Float` and `Integer` as aliases for `float` | `integer` (separate issue)

Reply:
I don't think we should do that, since those are not actually dtypes, but rather values to a `downcast` keyword which eg also accepts "signed"/"unsigned"